development/tests/test_match_details.py

```diff
@@ -1,9 +1,12 @@
-from unittest.mock import patch
 
+from unittest.mock import PropertyMock, patch
+from parameterized import parameterized
 from django.contrib.auth import get_user_model
 from django.test import TestCase
 
 from auth_app.models import Bot
+from development.common.logs_utils import FilterLogs
+from development.constants import ALL_ACTIONS, ALL_STATES, INVALID_STATE, MOVE_ACTION, SHOOT_ACTION, VALID_STATE
```

Comment:
Could you make it multiple lines and sorted alphabetically?

Reply:
Yes, of course